firefox-ios/firefox-ios-tests/Tests/ClientTests/ContentBlockerTests.swift

```diff
@@ -6,17 +6,23 @@ import XCTest
 @testable import Client
 
 final class ContentBlockerTests: XCTestCase {
+    var subject: ContentBlocker!
```

Comment:
Hmm, I believe this isn't accomplishing what you think it's accomplishing. 🤔  Since the ContentBlocker is a global singleton, setting it to nil in tearDown and back to the singleton on setUp is just resetting it to the same global instance each time (won't clear anything in a helpful way, if that was the intention).

If you want to verify for yourself, try putting a breakpoint on line 95 of ContentBlocker.swift. You'll see it only gets triggered once if you run `testCompileListsNotInStore_callsCompletionHandlerSuccessfully` 100 times. :) Or if you run this test suite (with 2 tests) once.

Reply:
you're right! good catch! In that case, I'll go ahead and disable this test as it proven to be flakey and global singletons are hard to test in general. Might open up to discussion on how we want to use this going forward.